java/src/jmri/implementation/NmraConsistManager.java

```diff
@@ -18,12 +19,12 @@ public NmraConsistManager() {
     }
 
     @Override
-    public Consist addConsist(DccLocoAddress address) {
+    public Consist addConsist(LocoAddress address) {
         if (consistTable.containsKey(address)) {
             return (consistTable.get(address));
         }
         NmraConsist consist;
-        consist = new NmraConsist(address);
+        consist = new NmraConsist((DccLocoAddress) address);
```

Comment:
Before this cast, you need to make sure the address is a DCCLocoAddress.  You should throw an illegal argument exception if it isn't.